tests/test_db.rs

```diff
@@ -907,6 +947,32 @@ fn test_open_cf_for_read_only() {
     }
 }
 
+#[test]
+fn test_open_cf_descriptors_for_read_only() {
+    let path = DBPath::new("_rust_rocksdb_test_open_cf_descriptors_for_read_only");
+    let cfs = vec!["cf1"];
+    {
+        let mut opts = Options::default();
+        opts.create_if_missing(true);
+        opts.create_missing_column_families(true);
+        let db = DB::open_cf(&opts, &path, cfs.clone()).unwrap();
```

Comment:
```suggestion
        let db = DB::open_cf(&opts, &path, &cfs).unwrap();
```
Redundant clone here.